contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
-   <ComparisonRow column1={true} column2={false} feature="Zapier" description="Trigger Zapier automations" />
-   <ComparisonRow column1={true} column2={false} feature="Sentry" description="Connect to Sentry data" />
-   <ComparisonRow column1={true} column2={false} feature="Zendesk" description="Two-way integration for customer support" />
-   <ComparisonRow column1={true} column2={true} feature="Slack" description="Alerts for Slack" />
-   <ComparisonRow column1={true} column2={true} feature="Microsoft Teams" description="Alerts for Microsoft Teams" />
-   <ComparisonRow column1={true} column2={false} feature="Community integrations" description="Build your own integration" />
-   <ComparisonRow column1={false} column2={true} feature="Google Ads" description="Import ROI data from Google Ads" />   
+   <ComparisonRow column1="1 million events" column2="50k tracked users" feature="Free analytics usage" description="Free tier usage each month" />
+   <ComparisonRow column1="5,000 recordings" column2="1,000 recordings" feature="Free replay usage" description="Free tier usage each month" />
+   <ComparisonRow column1="Unlimited" column2="10 only" feature="Saved insights" description="Number of you charts you can create" />   
+   <ComparisonRow column1={true} column2={false} feature="Custom events" description="Create custom tracking events" />
+   <ComparisonRow column1={true} column2={false} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
+   <ComparisonRow column1={true} column2={false} feature="Formulas" description="Create custom insights using formulas" />
+   <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Release and control features" />
+   <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Run tests on users and groups" />
+   <ComparisonRow column1={true} column2={false} feature="Surveys" description="Gather feedback and book user interviews" />
 </ComparisonTable>
 
-### Pricing
+- Amplitude offers a free Starter plan, but it has many arbitrary limitations and a low limit on usage.
 
-**PostHog** charges for actual usage – i.e. product analytics events, feature flag API requests, and so on. No credit card is required to signup and use PostHog, but entering a credit card unlocks almost all features and **you'll pay nothing** if you stay within our **generous free tier** limits:
+- Users who qualify for [our startup program](/startups) also get $50,000 in PostHog credit and a range of additional benefits. 
 
-|                       | **Free usage per month** 
-|-----------------------|--------------------------
-| **Product analytics** | 1 million events        
-| **Session replay**    | 5,000 recordings         
-| **Feature flags**     | 1 million API requests
-| **A/B testing**       | 1 million API requests  
-| **Surveys**           | 250 responses            
+### Analytics pricing
```

Comment:
This section is very difficult to read tbh. Feels like an entire blog in itself. Can we simplify it?

The whole explaining anonymous vs identified events is too much. Its a difficult concept to wrap my head around